Sources/Controllers/2.0/BottomSheet/WhatsNewBottomSheet/OAWhatsNewBottomSheetViewController.m

```diff
@@ -44,68 +47,59 @@ - (void) applyLocalization
 - (void) generateData
 {
     NSMutableArray *data = [NSMutableArray new];
-    
-    NSString *fullAppVersion = [[[NSBundle mainBundle] infoDictionary] objectForKey:@"CFBundleShortVersionString"];
-    NSString *title = [NSString stringWithFormat:OALocalizedString(@"latest_version"), fullAppVersion];
-    NSString *releaseNotesKey = [NSString stringWithFormat:@"ios_release_%@", [OAAppVersionDependentConstants getShortAppVersion]];
-    
     [data addObject:@[
         @{
              @"type" : kDescrTitleCell,
-             @"title" : title,
-             @"description" : OALocalizedString(releaseNotesKey)
         }]];
     _data = data;
 }
 
+- (NSMutableAttributedString *)getAttributedContentText
+{
+    NSString *fullAppVersion = [[[NSBundle mainBundle] infoDictionary] objectForKey:@"CFBundleShortVersionString"];
+    NSString *title = [NSString stringWithFormat:OALocalizedString(@"latest_version"), fullAppVersion];
+    NSString *releaseNotesKey = [NSString stringWithFormat:@"ios_release_%@", [OAAppVersionDependentConstants getShortAppVersion]];
+    NSString *description = OALocalizedString(releaseNotesKey);
+    
+    NSString *labelText = [NSString stringWithFormat:@"%@\n\n%@", title, description];
+    NSRange boldRange = NSMakeRange(0, title.length);
+    NSRange fullRange = NSMakeRange(0, labelText.length);
+    
+    NSMutableAttributedString *attributedString = [[NSMutableAttributedString alloc] initWithString:labelText];
+    NSMutableParagraphStyle *paragraphStyle = [[NSMutableParagraphStyle alloc] init];
+    [paragraphStyle setLineSpacing:5];
+    [paragraphStyle setLineHeightMultiple:0.8];
+    [attributedString addAttribute:NSParagraphStyleAttributeName value:paragraphStyle range:fullRange];
+    
+    UIFont *regularFont = [UIFont systemFontOfSize:17 weight:UIFontWeightRegular];
+    UIFont *semiboldFont = [UIFont systemFontOfSize:17 weight:UIFontWeightSemibold];
+    [attributedString addAttribute:NSFontAttributeName value:regularFont range:fullRange];
+    [attributedString addAttribute:NSFontAttributeName value:semiboldFont range:boldRange];
+    
+    return attributedString;
+}
+
+- (CGFloat) getViewHeight
+{
+    CGFloat width = DeviceScreenWidth - 2 * kHorisontelMargin;
```

Comment:
Again, this is incorrect, to change the inital anchor point, override the getInitialHeight method. Overriding getViewHeight will break scrolling.

Reply:
done